suite-native/module-home/src/components/BiometricsBottomSheet.tsx

```diff
@@ -0,0 +1,174 @@
+    backgroundColor: utils.colors.backgroundSurfaceElevation2,
+    color: utils.colors.iconPrimaryDefault,
+    width: IMG_WRAPPER_DIMENSIONS,
+    height: IMG_WRAPPER_DIMENSIONS,
+}));
+
+const textContentStyle = prepareNativeStyle(utils => ({
+    gap: utils.spacings.small,
+}));
+
+const buttonWrapperStyle = prepareNativeStyle(utils => ({
+    gap: utils.spacings.medium,
+    paddingHorizontal: utils.spacings.small,
+}));
+
+export const BiometricsBottomSheet = () => {
+    const { applyStyle } = useNativeStyles();
+    const { translate } = useTranslate();
+    const { showToast } = useToast();
+    const { hasDecidedBiometrics, setHasDecidedBiometrics } = useHasDecidedBiometrics();
+    const { isBiometricsOptionEnabled } = useIsBiometricsEnabled();
+
+    const [isVisible, setIsVisible] = useState(false);
+    const [biometricsTypes, setBiometricsTypes] = useState<AuthenticationType[]>([]);
+
+    const isFace = biometricsTypes?.filter(biometricsType =>
+        [AuthenticationType.FACIAL_RECOGNITION, AuthenticationType.IRIS].includes(biometricsType),
+    );
+
+    const iconName = isFace ? 'faceId' : 'fingerprint';
+
+    const title = useMemo(() => {
+        if (biometricsTypes?.length === 0) {
+            return <Translation id="moduleHome.biometricsModal.title.unknown" />;
+        }
+        if (isFace) {
+            return <Translation id="moduleHome.biometricsModal.title.faceId" />;
+        }
+        if (Platform.OS === 'android') {
+            return <Translation id="moduleHome.biometricsModal.title.fingerprint" />;
+        }
+        return <Translation id="moduleHome.biometricsModal.title.touchId" />;
+    }, [biometricsTypes, isFace]);
+
+    useEffect(() => {
+        const getSupportedTypes = async () => {
+            const biometricsTypesAvailable = await supportedAuthenticationTypesAsync();
+            setBiometricsTypes(biometricsTypesAvailable);
+        };
+        getSupportedTypes();
+    }, []);
+
+    useEffect(() => {
+        let isMounted = true;
+        let timerId: ReturnType<typeof setTimeout>;
+        const checkBiometrics = async () => {
+            const isBiometricsAvailable = await getIsBiometricsFeatureAvailable();
+            if (isBiometricsAvailable && !isBiometricsOptionEnabled) {
+                timerId = setTimeout(() => {
+                    if (isMounted) {
+                        setIsVisible(true);
+                    }
+                }, SHOW_TIMEOUT);
+            }
+        };
+        checkBiometrics();
+
+        return () => {
+            clearTimeout(timerId);
+            isMounted = false;
+        };
+    }, [isBiometricsOptionEnabled, hasDecidedBiometrics]);
+
+    const handleClose = () => {
+        analytics.report({
+            type: EventType.ModalBiometricsDecision,
+            payload: { enabled: false },
+        });
+        setIsVisible(false);
+        setHasDecidedBiometrics(true);
+    };
+
+    const handleEnable = async () => {
```

Comment:
Please - this should be unified with `toggleBiometricsOption` from `suite-native/module-settings/src/screens/SettingsPrivacyAndSecurity.tsx`

This should be reusable and enabling and disabling should have same outcome in both places

Reply:
updated in 3f2db7d